cubic-server/types.cpp

```diff
@@ -35,3 +36,11 @@ u128 u128::fromShortString(const std::string &str)
     std::string second = std::string(it, str.end());
     return u128({std::stoull(first, 0, 16), std::stoull(second, 0, 16)});
 }
+
+u128 u128::random()
+{
+    std::random_device rd;
+    std::mt19937_64 gen(rd());
```

Comment:
Please create the `std::random_device` and the `std::mt19937_64` for caching, it is unnecessary to create them every single time and defeats the purpose of randomness.